.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,14 @@
+blank_issues_enabled: false
+contact_links:
+  - name: AICA Documentation
+    url: https://docs.aica.tech
+    about: Find more developer information on the AICA documentation page
+  - name: AICA Website
+    url: https://aica.tech
+    about: Find more product information on the AICA website
```

Comment:
What are lines 9 to 14 for? Why refer to community and bug bounty?

Reply:
Yes good question. When I started testing issue templates [more than a year ago](https://github.com/eeberhard/test-repository/commit/8fa9cb6c0b84990994483e177f38ab983f2ce82e), every issue template included GitHub community issues and bug bounty links by default. I think I remember that if I didn't explicitly specify them in the config, they would appear anyway but they would be at the top of the issue submission form. By adding this config they ended up at the bottom of the list.

But, I tested removing them again now and it seems like they aren't included by default anymore.